hack/verify-style.sh

```diff
@@ -20,6 +20,10 @@ if [[ ! $(which operator-sdk) ]]; then
   echo "operator-sdk not found on PATH. To install:"
   echo "go get -u github.com/operator-framework/operator-sdk/commands/operator-sdk"
 fi
+if [[ ! $(which dep) ]]; then
```

Comment:
while you're adding this, I'm pretty sure you can remove the check for operator-sdk just above, can't you?

Reply:
definitely - especially since we don't `exit 1` anyways.